src/coreclr/vm/precode.h

```diff
@@ -52,7 +52,7 @@ EXTERN_C VOID STDCALL PrecodeRemotingThunk();
 
 #elif defined(TARGET_WASM)
 
-#define SIZEOF_PRECODE_BASE         0
+#define SIZEOF_PRECODE_BASE         8 // 2*pointer size
```

Comment:
Can we use a pointer size define so this won't break when built for memory64 later?